contracts/v2.6.2/src/hello.cairo

```diff
@@ -25,5 +26,9 @@ mod hello {
         fn set_name(ref self: ContractState, name: felt252) {
             self.name.write(name);
         }
+
+        fn sum(ref self: ContractState, a: felt252, b: felt252) -> felt252 {
```

Comment:
There is no modification of state, so you can use a snapshot instead:
```cairo
fn sum(self: @ContractState, a: felt252, b: felt252) -> felt252 {
```

Reply:
good point! thx